src/components/ArticleSummary.jsx

```diff
@@ -0,0 +1,12 @@
+export default function ArticleSummary({ data, }) {
+  return <article className="article-summary">
+    <div className="title">{data.title}</div>
+    <div className="details">
+      <div className="topic">{data.topic}</div>
+      <div className="votes">{data.votes}</div>
+      <div className="author">By {data.author}</div>
+      <div className="created_at">Posted: {data.votes}</div>
```

Comment:
Created_at has the value of data.votes here?

For some reason, all of the votes are appearing on the site as 0 - sounds like a back end issue. Likely not worth fixing right now (while the focus is on front end), but worth keeping in mind!